application/src/main/kotlin/application/ExamplesCommand.kt

```diff
@@ -174,12 +176,25 @@ For example:
         )
         var filterNot: List<String> = emptyList()
 
-        @Option(names = ["--contract-file"], description = ["Contract file path"], required = true)
-        lateinit var contractFile: File
+        @Option(names = ["--contract-file", "--spec-file"], description = ["Contract file path"], required = false)
+        var contractFile: File? = null
 
         @Option(names = ["--example-file"], description = ["Example file path"], required = false)
         val exampleFile: File? = null
 
+        @Option(names = ["--examples-dir"], description = ["Examples directory path associated to a single spec"], required = false)
+        val examplesDir: File? = null
+
+        @Option(names = ["--specs-dir"], description = ["Specs directory path"], required = false)
+        val specsDir: File? = null
+
+        @Option(
+            names = ["--examples-base-dir"],
+            description = ["Examples directory path containing multiple example directories associated to multiple specs"],
```

Comment:
Base directory which contains multiple external examples directories each named as per the Specmatic naming convention to associate them with the corresponding API specification

Reply:
Addressed